lib/logstash/outputs/datadog_metrics.rb

```diff
@@ -9,8 +10,7 @@
 
 # Default `queue_size` and `timeframe` are low in order to provide near realtime alerting.
 # If you do not use Datadog for alerting, consider raising these thresholds.
-
-class LogStash::Outputs::DatadogMetrics < LogStash::Outputs::Base
+module LogStash module Outputs class DatadogMetrics < LogStash::Outputs::Base
```

Comment:
While you were away I added this to the redis input via a merged PR. (I think the discussion was to do it incrementally).  One has to be a bit careful when inside the class after making this change.